services/workflows-service/src/env.ts

```diff
@@ -25,6 +25,7 @@ export const env = createEnv({
     PORT: z.coerce.number(),
     DB_URL: z.string().url(),
     SESSION_SECRET: z.string(),
+    API_KEY_SECRET: z.string(),
```

Comment:
This env variable name doesn't explain well what it does. Consider renaming it. Maybe `HASHING_KEY` or something.

Reply:
changed to : `HASHING_KEY_SECRET`